pinot-server/src/main/java/org/apache/pinot/server/starter/helix/HelixInstanceDataManager.java

```diff
@@ -180,6 +181,24 @@ private void initInstanceDataDir(File instanceDataDir) {
     }
   }
 
+  @Override
+  public List<File> getConsumerDirPaths() {
+    String consumerDirPath = _instanceDataManagerConfig.getConsumerDir();
+
+    if (consumerDirPath != null) {
+      return Collections.singletonList(new File(consumerDirPath));
+    } else {
```

Comment:
this is when consuming and completed dir is mixed up right? could we add some comment to the code?

Reply:
updated to use `RealtimeTableDataManager` for this logic which has a comment about this.